edit_code_utils.py

```diff
@@ -30,14 +30,14 @@ def get_function_code(function_name, file_name):
 
     function_def = re.search(r'\n\s*def\s+' + function_name + r'\(', code, re.MULTILINE)
     if not function_def:
-        raise Exception("Could not find function definition for " + function_name)
+        raise Exception(f"Could not find function definition for {function_name}")
```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer. Leave feedback on this recommendation by replying to the comment or by reacting to the comment using emoji.

Using `Exception` and `BaseException` can make your code prone to errors and difficult to maintain. Instead, we recommend using one of the [Built-in Exceptions](https://docs.python.org/3/library/exceptions.html) or creating a custom exception class that is derived from `Exception` or one of its subclasses.